### DISCUSSION TITLE
Migrate my chrome license to Firefox

### DISCUSSION THREAD

**Discussion by wenm89** [aclap-dev/video-downloadhelper#1403] [License & Purchase requests]
### What's your issue?

Hi,

Please migrate my Chrome license to Firefox.

### The 10 first digits (only 10) of your license number or email address

3440d5dce7

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Ahmedsidala** [comment]
I would like to transfer my Chrome license to Firefox, please
My first 10 digits of the license are as follows: 3160a409bf

**Comment by Ahmedsidala** [comment]
Please migrate my Chrome license to Firefox.

The 10 first digits (only 10) of your license number or email address
3160a409bf

I confirm that I did not paste my full license number
 I confirm

**Comment by jcfrog** [comment]
Hi, Sorry we can't swap licenses. Chrome and Firefox premium are separate products with different prices and features.

**Comment by Ahmedsidala** [comment]
The program is no longer downloading from YouTube on chrome. But working on Firefox, so Is there a solution for this at least?

Sent from my iPhone
Best regards,
A. Shalash

On Oct 30, 2024, at 11:30 AM, jcfrog ***@***.***> wrote:

﻿

Hi, Sorry we can't swap licenses. Chrome and Firefox premium are separate products with different prices and features.

**Reply by jcfrog** [reply]
Chrome extension never allowed Youtube downloads, that's why Chrome premium is cheaper.

**Comment by Ahmedsidala** [comment]
It used to allow it. I did downloads there myself few months ago.

Sent from my iPhone
Best regards,
A. Shalash

On Oct 30, 2024, at 8:49 PM, jcfrog ***@***.***> wrote:

﻿

Chrome extension never allowed Youtube downloads, that's why Chrome premium is cheaper.
